core/src/wallet/stellar/StellarLikeAccount.cpp

```diff
@@ -253,11 +253,20 @@ namespace ledger {
         }
 
         void StellarLikeAccount::interpretTransaction(const stellar::Transaction &tx, std::vector<Operation> &out) {
+            auto log = logger();
             if (tx.envelope.type != stellar::xdr::EnvelopeType::ENVELOPE_TYPE_TX_V0 &&
                 tx.envelope.type != stellar::xdr::EnvelopeType::ENVELOPE_TYPE_TX) {
                 // Ignore transaction with unhandled envelope types.
                 return;
             }
+            // Ignore transaction if at least one operation has an unsupported type
+            const auto& operations = stellar::xdr::getOperations(tx.envelope);
+            for (const auto& op : operations) {
```

Comment:
For later, as it's not the goal of this PR

We could avoid skipping the entire transaction by just skipping the problematic operation with an early `continue` l.315

Reply:
Not really I believe, because if we crossed an unknown operation during decoding it means we probably didn't how much binary data followed the operation and our binary read cursor is now lost and all following operations are lost.

It's true we could do a best effort in keeping all operations _until_ we meet one we don't know, but then we'd end up with half parsed transactions which is arguably worse than just a all-or-nothing approach.